test/inductor/test_torchinductor_codegen_dynamic_shapes.py

```diff
@@ -156,6 +156,7 @@ def run(*ex, **kwargs):
     "test_views3_dynamic_shapes": TestFailure(("cpu",)),
     "test_views4_dynamic_shapes": TestFailure(("cpu",)),
     "test_zeros_dynamic_shapes": TestFailure(("cpu",)),
+    "test_upsample_cat_conv_dynamic_shapes": TestFailure(("cpu",)),
```

Comment:
Why skip?

Reply:
there has format conversion for our customer op(nchw->nhwc, C is a const value):
```
            }
            #pragma GCC ivdep
            for(long i1=static_cast<long>(0); i1<static_cast<long>(8); i1+=static_cast<long>(1))
            {
                #pragma GCC ivdep
                for(long i2=static_cast<long>(0); i2<static_cast<long>(4*ks2*ks3); i2+=static_cast<long>(1))
                {
                    auto tmp0 = in_ptr2[static_cast<long>(i2 + (4*i1*ks2*ks3) + (32*i0*ks2*ks3))];
                    out_ptr2[static_cast<long>(i1 + (8*i2) + (32*i0*ks2*ks3))] = tmp0;
                }
            }
```

Reply:
Seems the test checker is too restrict. Not necessarily all the for loops have dynamic bound. We may consider to relax it in a separate PR.